### PR TITLE
Clarified metadata location options and added note on tradeoffs

### PR DESCRIPTION
**Pull request by dbooth-boston**

### CODE REVIEW

**Comment by gkellogg** [comment]
Of these, the only thing I think we should consider is 4674ce9dccbf22fbaacaf2cc7fb9789868d09118; see comments on efa2d676a295c4717e29296f9a3a8b2376275b60.

So -1 for merge.

**Comment by iherman** [comment]
As, essentially, expressed elsewhere:

https://lists.w3.org/Archives/Public/public-csv-wg/2015Oct/0026.html https://lists.w3.org/Archives/Public/public-csv-wg/2015Oct/0026.html

-1 to this PR

Ivan

> On 28 Oct 2015, at 23:26, David Booth notifications@github.com wrote:
> 
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/w3c/csvw/pull/768 https://github.com/w3c/csvw/pull/768
> Commit Summary
> 
> Clarified metadata location options and added note on tradeoffs
> Clarified metadata location options and added note on tradeoffs
> Clarified metadata location options and added note on tradeoffs
> File Changes
> 
> M syntax/index.html https://github.com/w3c/csvw/pull/768/files#diff-0 (9)
> Patch Links:
> 
> https://github.com/w3c/csvw/pull/768.patch https://github.com/w3c/csvw/pull/768.patch
> https://github.com/w3c/csvw/pull/768.diff https://github.com/w3c/csvw/pull/768.diff
> —
> Reply to this email directly or view it on GitHub https://github.com/w3c/csvw/pull/768.

---

Ivan Herman, W3C
Digital Publishing Lead
Home: http://www.w3.org/People/Ivan/
mobile: +31-641044153
ORCID ID: http://orcid.org/0000-0003-0782-2704

File: syntax/index.html [hunk 1]
```diff
@@ -493,7 +491,7 @@
         </div>
       </section>
       <section>
-        <h3>Site-wide Location Configuration</h3>
+        <h3>Site-wide Location Configuration and Standard Metadata Filenames</h3>
```
Comment (JeniT) [inline syntax/index.html:494]: I'd be happy to accept a PR with just this change in it.

Comment (iherman) [inline syntax/index.html:494]: Grudgingly ok with this, if everybody does. (We introduce a new notion into the standard that was not the intention, but let us close things.)

Ivan

---

Ivan Herman
Tel:+31 641044153
http://www.ivan-herman.net

(Written on mobile, sorry for brevity and misspellings...)

**Comment by iherman** [comment]
Closed the PR without accepting it. The documents are now in Proposed Recs, cannot be changed any more.

**Pull request closed without merging**